dataset/dictionary/story.json5

```
@@ -820,4 +820,11 @@
     ja: "降臨者",
     zhCN: "降临者",
   },
+  {
+    en: "Magic Tonic", // TODO: v4.8 期間限定イベントの折り紙キャラクターの飲み物。魔法のインク
+    ja: "魔水",
+    zhCN: "魔水",
+    pronunciationJa: "ますい",
+  },
```

```suggestion
    tags: [ "event" ],
  },
```

Let's add `event` tag here.